openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1059,7 +1059,7 @@ The following parameters are used in the Batch Credential Request:
 
 * `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request).
 
-Below is a non-normative example of a Batch Credential Request:
+Below is a non-normative example of a Batch Credential Request for two Credentials in an [@VC_DATA] format and [@ISO.18013-5] format:
```

Comment:
I find this sentence hard to read, suggest the following

```suggestion
Below is a non-normative example of a Batch Credential Request for two Credentials; one in [@VC_DATA] format and another in [@ISO.18013-5] format:
```

Reply:
for the future guidance.. for editorial suggestions, would really appreciate approval conditional to suggested changes to being accepted. if I agree with the changes, I can just accept them and merge a PR without asking to re-review, and if i do not agree, I will reach back to you to clarify until we reach agreement.